ui/app/src/services/content.ts

```diff
@@ -24,7 +24,7 @@ import { LookupTable } from '../models/LookupTable';
 import $ from 'jquery/dist/jquery.slim';
 import { dataUriToBlob, isBlank, popPiece, removeLastPiece } from '../utils/string';
 import ContentInstance from '../models/ContentInstance';
-import { AjaxResponse } from 'rxjs/ajax';
+import { ajax, AjaxResponse } from 'rxjs/ajax';
```

Comment:
Is this still used? Check console warnings (compile/build).

Reply:
this one is not, removing